### PR TITLE
ansible: initial integration

### PR DESCRIPTION
**Pull request by arthurscchan**
initial fuzzer setup

### CODE REVIEW

File: projects/ansible/playtask_fuzz.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+def TestInput(input_bytes):
+   fdp = atheris.FuzzedDataProvider(input_bytes)
+
+   try:
+      task1 = Task.load({'name': fdp.ConsumeString(10), 'shell': fdp.ConsumeString(10), 'action': fdp.ConsumeString(10)})
+      task2 = Task.load({'name': fdp.ConsumeString(10), 'shell': fdp.ConsumeString(10), 'action': fdp.ConsumeString(10)})
+      task3 = Task.load({'action': fdp.ConsumeString(10)})
+
+      Play.load(dict(name=fdp.ConsumeString(10),hosts=[fdp.ConsumeString(5)],gather_facts=fdp.ConsumeBool(),tasks=[task1,task2,task3]))
+   except Exception:
```
Comment (DavidKorczynski) [inline projects/ansible/playtask_fuzz.py:31]: Can we specify this more than `Exception`? I would assume the above functions are not meant to be able to throw any arbitrary exception?

**Review by DavidKorczynski — CHANGES_REQUESTED** [review]

File: projects/ansible/playtask_fuzz.py (hunk 1 above)
Comment (arthurscchan) [reply projects/ansible/playtask_fuzz.py]: Replaced with ansible specific exception

**Comment by DavidKorczynski** [comment]
This PR supersedes https://github.com/google/oss-fuzz/pull/4923

File: projects/ansible/task_fuzz.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import atheris
+with atheris.instrument_imports():
+   from ansible.errors import AnsibleError, AnsibleParserError, AnsibleUndefinedVariable
+   from ansible.playbook.task import Task
+
+def TestInput(input_bytes):
+   #for temp passing CI only
```
Comment (oliverchang) [inline projects/ansible/task_fuzz.py:22]: what is this for?

File: projects/ansible/build.sh [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+# Build and install project (using current CFLAGS, CXXFLAGS).
+pip3 install --upgrade pip
+pip3 install -r ./requirements.txt
+pip3 install .
+
+# Build fuzzers in $OUT.
+cd $SRC
+compile_python_fuzzer parse_fuzz.py --add-data ansible/lib/ansible/config:ansible/config
+#compile_python_fuzzer task_fuzz.py --add-data ansible/lib/ansible/config:ansible/config
```
Comment (oliverchang) [inline projects/ansible/build.sh:26]: Should this still be commented?

**Review by oliverchang — APPROVED** [review]
nice!

**Review by DavidKorczynski — APPROVED** [review]

File: projects/ansible/build.sh (hunk 3 above)
Comment (arthurscchan) [reply projects/ansible/build.sh]: fixed

File: projects/ansible/task_fuzz.py (hunk 2 above)
Comment (arthurscchan) [reply projects/ansible/task_fuzz.py]: fixed

**Pull request merged**